features/modified_execution_paths/change_in_branch_spec.rb

```diff
@@ -8,6 +8,16 @@
   include_context 'model1 examples'
   include_context 'base forecast'
 
+  let(:map_generator_config) do
+    <<~CONFIG
+      Crystalball::MapGenerator.start! do |c|
+        c.register Crystalball::MapGenerator::CoverageStrategy.new
+        c.register Crystalball::MapGenerator::DescribedClassStrategy.new
+        c.register Crystalball::MapGenerator::AllocatedObjectsStrategy.build(only: ['Object'])
```

Comment:
I think it is not necessary to have all of these here. We can stay with `CoverageStrategy` only